README.md

```diff
@@ -1,11 +1,12 @@
-# Project Name
+# Javasqript Pizzeria
 
-Replace this readme with your own information about your project. Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Assignment to create a simple javasqript pizzeria where costumers can choose and order different dishes.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Tried to do it as simple as possible using alerts, promts and conditional statements. I also made a version using switch.
+If I had more time I would try to make the code less repetative and easier to read.
+I had a problem with the age promt where the user can leave the input field empty, press ok and still get a confirmation. If I had more time I would try to solve that.
```

Comment:
that's okay, you can always go back to your projects and improve them at a later stage. Very good you updated your README.md 👌